cpp/oneapi/dal/data/table.hpp

```diff
@@ -46,13 +46,18 @@ class table {
     std::int64_t get_row_count() const;
     const table_metadata& get_metadata() const;
 
+    std::int64_t get_kind() const {
+        return pure_empty_table_kind_;
```

Comment:
Seems like it's not rights implementation: `table::get_kind` should request kind from its pimpl.

Reply:
It will be good to have very simple test case:
```c++
table x = homogen_table{...};
ASSERT_EQ(x.get_kind(), homogen_table::kind());
```

Reply:
Thanks, fixed